scripts/dts/edtsdatabase.py

```diff
@@ -0,0 +1,257 @@
+#
```

Comment:
Can you pull edtsdatabase.py out into its own commit, so we have a set of commits that are related just to EDTS (the creation of the json and the parsing).

Reply:
@galak , Done: #9876